dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc

```diff
@@ -591,7 +591,7 @@ EOD;
 						$rndc_conf_path = BIND_LOCALBASE . "/etc/rndc.conf";
 						$rndc = "/usr/local/sbin/rndc -q -c $rndc_conf_path";
 						if ($zone_is_dynamic) {
-							exec("$rndc freeze $zonename IN $zoneview");
+							exec("$rndc freeze " . escapeshellargs($zonename) . " IN " . escapeshellargs($zoneview));
```

Comment:
These should all be `escapeshellarg()`, not `escapeshellargs()` (no `s` on the end)

Reply:
:flushed: I guess I get what I deserve for trying to "fix a small issue" when I'm not in front my my main workstation. Correct fixing commit pushed already.